packages/queued-request-controller/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/queued-request-controller",
-  "version": "0.12.0",
+  "version": "1.0.0",
```

Comment:
oooo he did it. Stepping into the major leagues 🤣

Reply:
haha can change this back to minor if preferred

Reply:
Nah its time for this boy to become a man